core/controllers/galleries.py

```diff
@@ -75,6 +75,27 @@
     }])
 
 
+def get_explorations_list(query_string, search_cursor):
+    """Handles getting exploration list"""
```

Comment:
Given a query string and a search cursor, returns a list of explorations that satisfy the search query.